dlt/common/schema/utils.py

```diff
@@ -698,4 +698,4 @@ def standard_hints() -> Dict[TColumnHint, List[TSimpleRegex]]:
 
 
 def standard_type_detections() -> List[TTypeDetections]:
-    return ["iso_timestamp"]
+    return ["iso_timestamp", "iso_date"]
```

Comment:
this is the only thing I'd change. I'm not sure that generating dates should be the default. date-like strings are simple and common - IMO automatic conversion to `date` database type is an unexpected behavior.

instead you can always use `add_type_detection` - from within a dlt.source like here https://dlthub.com/docs/general-usage/schema#schema-is-modified-in-the-source-function-body

or just before loading: `source().schema.add_type_detection("iso_date")`.

we'll be releasing dlt soon so if you are OK with it we can do this change here

Reply:
I'm fine with that :)

Reply:
[91b7293](https://github.com/dlt-hub/dlt/pull/767/commits/91b729308b07c08b5b7578907a5ff387aa74782f)